src/lwc/gsApplicationStatus/gsApplicationStatus.css

```diff
@@ -0,0 +1,71 @@
+.container {
+    padding-left: 24px;
+    padding-top: 12px;
+    padding-right: 24px;
+    background-color: white;
+    border: solid 1px #b5b5b5;
+    box-shadow: 0px 6px 3px 0px #b5b5b5;
+    text-align: center;
+    
+}
+
+.image-container {
+    text-align: center;
+    width: 100%;
+    margin-bottom: 24px;
+}
+.image {
+    object-fit: fill;
+}
+
+.info {
+    text-align: center;
+}
+
+.info p {
+    margin-bottom: 18px;
+}
+
+.info .slds-button {
+   
+    width: 100%;
+    font-size: 15px;
+    margin-bottom: 24px;
+    padding: 10px;
+}
+
+.slds-button_outline-brand {
+    border-color: purple;
```

Comment:
Why override the button colors? Doesn't this violate SLDS design patterns, plus potentially create accessibility issues with contrast between the background color and text color? There is a reason that we try to stick with standard SLDS styles 99.9% of the time.

Reply:
Regarding this comment, We asked Mary Heid about this and she answered: 

> There are two questions at play here, I believe.
> #1 - does the color meet guidelines? Yes.
> #2 - SLDS design patterns - <I’m not an SLDS developer, but> if we use the SLDS design tokens, when those update, we get those updates for ‘free.’ For example, I believe the the SLDS team is working to update tokens to meet the newer WCAG 2.1 guidelines. While that’s not an issue in this particular use case, it speaks to the point and value of using those tokens. Additionally, I believe the tokens inherently support accessibility features such as inverted colors when a user invokes high contrast mode in their browser/os..

The color does not violates any color guidelines but still has the cons of not having an specific design-token for it, the design was previously approved  (you can talk to Stephanie Zeitz about it), so I would stick to the purple